govdelivery/tests/utils.py

```diff
@@ -0,0 +1,11 @@
+import codecs
+from os import path
+
+
+testdata_dir = path.join(path.dirname(__file__), 'test_api_responses')
+
+
+def load_data(filename):
+    full_path = path.join(testdata_dir, filename)
+    f = codecs.open(full_path, encoding="utf8")
+    return f.read()
```

Comment:
On Python 3 this code generates warnings like `ResourceWarning: unclosed file <_io.BufferedReader name='govdelivery/govdelivery/tests/test_api_responses/list_all_topics.xml'>` because the files are left open. To close, try something like:

```py
with codecs.open(full_path, encoding="utf-8") as f:
    return f.read()
```

(Note the change to [`"utf-8"`](https://docs.python.org/3/library/codecs.html#standard-encodings).)

Reply:
Fixed in 88ee0a6